catalog/db_diff_scanner.go

```diff
@@ -82,44 +85,48 @@ func (s *DiffScanner) diffFromParent(tx db.Tx, params doDiffParams) (*DiffScanne
 	if err != nil {
 		return nil, fmt.Errorf("get child last commit failed: %w", err)
 	}
-	scannerOpts := DBScannerOptions{
-		After:            params.After,
-		AdditionalFields: prepareDiffAdditionalFields(params.AdditionalFields),
+	leftLineage, err := getLineage(tx, params.LeftBranchID, CommittedID)
+	if err != nil {
+		return nil, err
```

Comment:
use fmt.Errorf to describe the error as it was from getting lineage of the left branch